tests/x11/firefox_audio.pm

```diff
@@ -18,16 +18,18 @@ use testapi;
 
 sub run {
     my ($self) = @_;
-    $self->start_firefox();
-    send_key "ctrl-l";
-    type_string(autoinst_url . "/data/1d5d9dD.oga");
-    send_key "ret";
     start_audiocapture;
```

Comment:
Before we started capturing audio after navigating to the url, here sequence is different. Does it make any difference?